### ISSUE TITLE
ColorColumn has to be cleared manually

### ISSUE THREAD

**Issue by wimstefan** [lukas-reineke/virt-column.nvim#2]
Well the title says it all already ...
My current configuration:
```lua
  use {
    'lukas-reineke/virt-column.nvim',
    event = 'BufRead',
    config = function()
      vim.opt.colorcolumn = '81'
      require('virt-column').setup()
      vim.cmd[[highlight clear ColorColumn]]
    end
  }
```
The result looks like this:
![scrot-2021-11-27-14:37](https://user-images.githubusercontent.com/4605934/143683794-0436d43d-42e0-43a1-8171-f70e40b38750.png)

Only if I execute `highlight clear ColorColumn` in the commandline I get a nice tiny line instead of the `ColorColumn`

**Comment by mcauley-penney** [comment]
I placed the command in the setup function itself and it works

```lua
    use
    {
        use "lukas-reineke/virt-column.nvim",
        require "virt-column".setup{
            char = "│",
            vim.cmd "highlight clear ColorColumn"
        }
    } 
```

![image](https://user-images.githubusercontent.com/59481467/143777362-e2ad96c9-2592-4b16-b4a3-46fa420ef3ff.png)

**Comment by lukas-reineke** [comment]
Yeah it is mentioned in the README. I thought some users might want to use both.
But it is probably more common to disable it, and the plugin should just do it.
I'll add it

**Comment by lukas-reineke** [comment]
@mcauley-penney thank you :+1: but there are a couple of things in your code that you should fix.

First, you call `use` twice, you don't need that.

Second, the setup call should be inside a config function.

And then lastly, `vim.cmd` clears the highlight, that's already everything you need. But then you take the return value (which is `nil`) and add it into the options.
Just run it outside the setup table

It should look like this

```lua
use {
	"lukas-reineke/virt-column.nvim",
	config = function()
		vim.cmd "highlight clear ColorColumn"
		require "virt-column".setup {
			char = "│",
		}
	end
} 
```

**Comment by mcauley-penney** [comment]
Thanks! The second use was a typo, thanks for catching that

**Comment by wimstefan** [comment]
Thank you @mcauley-penney for your help & tips 👍
And most certainly thanks to @lukas-reineke for diving into the issue and  implementing a fix! 👍👍👍

But the problem persisted :)
Until I started thinking a bit more about what could cause the highlighting and my first thought of course was the colorscheme. Which is loaded by `lualine`. As soon as I configured `virt-column.nvim` to be loaded **after** `lualine` the problem was gone :)

Thanks again guys for your support 🙏
